jwcrypto/jwk.py

```diff
@@ -549,12 +550,84 @@ def import_from_pyca(self, key):
         else:
             raise InvalidJWKValue('Unknown key object %r' % key)
 
+    def import_from_pem(self, data, password=None):
+        """Imports a key from data loaded from a PEM file.
+        The key may be encrypted with a password.
+        Private keys (PKCS#8 format), public keys, and X509 certificate's
+        public keys can be imported with this interface.
+
+        :param data(bytes): The data contained in a PEM file.
+        :param password(bytes): An optional password to unwrap the key.
+        """
+
+        try:
+            key = serialization.load_pem_private_key(
+                data, password=password, backend=default_backend())
+        except ValueError as e:
+            if password is not None:
+                raise e
+            try:
+                key = serialization.load_pem_public_key(
+                    data, backend=default_backend())
+            except ValueError:
+                try:
+                    cert = x509.load_pem_x509_certificate(
+                        data, backend=default_backend())
+                    key = cert.public_key()
+                except ValueError:
+                    raise e
+
+        self.import_from_pyca(key)
+
+    def export_to_pem(self, private_key=False, password=False):
+        """Exports keys to a data buffer suitable to be stored as a PEM file.
+        Either the public or the private key can be exported to a PEM file.
+        For private keys the PKCS#8 format is used. If a password is provided
+        the best encryption method available as determined by the cryptography
+        module is used to wrap the key.
+
+        :param private_key: Whether the private key should be exported.
+         Defaults to `False` which means the public key is exported by default.
+        :param password(bytes): A password for wrapping the private key.
+         Defaults to False which will cause the operation to fail. To avoid
+         encryption the user must explicitly pass None, otherwise the user
+         needs to provide a password in a bytes buffer.
+        """
+        e = serialization.Encoding.PEM
+        if private_key is True:
```

Comment:
Won't this always match anything that is not "False" or "None", I want to make sure there is a True there and no a bytes blob with a private key for example.